### PR TITLE
[SW-1305] Move ability to change behavior of MOJO models to MOJOLoader

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Comment by jakubhava** [comment]
@mn-mikke we need to also update docs http://docs.h2o.ai/sparkling-water/2.4/latest-stable/doc/deployment/load_mojo.html?highlight=createfrommojo

http://docs.h2o.ai/sparkling-water/2.4/latest-stable/doc/deployment/scoring_mojo_pipeline.html?highlight=createfrommojo

File: ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala [hunk 1]
```diff
@@ -106,10 +108,11 @@ class H2OMOJOPipelineModelTest extends FunSuite with SparkTestContext {
     // Test data
     val df = spark.read.option("header", "true").csv("examples/smalldata/prostate/prostate.csv")
     // Test mojo
+    val mojoSettings = H2OMOJOSettings(namedMojoOutputColumns = false)
```
Comment (jakubhava) [inline ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala]: should be `namedMojoOutputColumns = false`

Comment (mn-mikke) [reply ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala]: Good catch, thanks!

Comment (jakubhava) [reply ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala]: I missed that during review, sorry

Comment (mn-mikke) [reply ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala]: There is no reason to apologize. It was my mistake ;-)

**Pull request merged**